src/new/sendTransaction.ts

```diff
@@ -572,14 +572,7 @@ export async function prepareSendTokensData(
       throw new Error(`Token: ${token}. Insufficient amount of tokens to fill the amount.`);
     }
     newtxData.inputs = newUtxos.utxos.map((utxo) => {
-      return {
-        txId: utxo.txId,
-        index: utxo.index,
-        value: utxo.value,
-        authorities: utxo.authorities,
-        token: utxo.token,
-        address: utxo.address,
-      } as IDataInput;
+      return helpers.getDataInputFromUtxo(utxo);
```

Comment:
Here we can just do:
```js
newTxData.inputs = newUtxos.utxos.map(helpers.getDataInputFromUtxo);
```

Reply:
Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/569/commits/527d4366e062f205de5d2f952338ae8302e7d561